app/scripts/views/ready.js

```diff
@@ -9,18 +9,54 @@
 
 define([
   'cocktail',
-  'views/form',
-  'stache!templates/ready',
-  'lib/url',
   'lib/constants',
-  'views/mixins/service-mixin',
+  'lib/url',
+  'stache!templates/ready',
+  'views/form',
   'views/marketing_snippet',
-  'views/marketing_snippet_ios'
+  'views/marketing_snippet_ios',
+  'views/mixins/service-mixin'
 ],
-function (Cocktail, FormView, Template, Url, Constants, ServiceMixin,
-      MarketingSnippet, MarketingSnippetiOS) {
+function (Cocktail, Constants, Url, Template, FormView, MarketingSnippet,
+  MarketingSnippetiOS, ServiceMixin) {
   'use strict';
 
+  function t(msg) {
+    return msg;
+  }
+
+  function generateHeader (id, text) {
+    return '<h1 id="' + id + '">' + text + '</h1>';
+  }
+
+  /*eslint-disable camelcase*/
+
+  // The "header" and "ready to sync" messages are generated
```

Comment:
See note: https://github.com/mozilla/fxa-content-server/pull/3094/files#r41073115

> One template per screen -

I like less HTML

> One module per screen 

this might work if we can extend "ready" properly

> Pass the strings in from router.js 

Not sold on doing that, seems like if we add a bit more it won't scale.

The current approach should be okay for now, just not a fan of generating that html in the JS file (per my comment)